devops/resources/add_new_xqueue.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -exuo pipefail
+
+# Required by click http://click.pocoo.org/5/python3/
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+cd $WORKSPACE/configuration/util/jenkins/add_new_xqueues
+pip install -r requirements.txt
+env
+
+. ../assume-role.sh
+assume-role ${ROLE_ARN}
+python ./add_xqueue.py --environment ${ENVIRONMENT} --deploy ${DEPLOYMENT}
```

Comment:
Can you change the naming of add_new_xqueue and add_queue.py to be 'add_xqueue_to_dashboard'